qiskit/execute_function.py

```diff
@@ -75,7 +75,25 @@ def execute(
     """Execute a list of :class:`qiskit.circuit.QuantumCircuit` or
     :class:`qiskit.pulse.Schedule` on a backend.
 
-    The execution is asynchronous, and a handle to a job instance is returned.
+    This function provides a higher level abstraction for applications where the details
+    of how the circuit is compiled and run on the backend don't matter and all that is
+    required is that you want to execute a :class:`~.QuantumCircuit` on a given backend.
+    The options on this function are minimal by design, if you desire more control over
+    the compilation of the circuit prior to execution you should instead use the
+    :class:`~.transpile` function with the :class:`.BackendV2.run` method to execute the
+    circuit. For example::
+
+        from qiskit.providers.fake_provider import FakeKolkata
+        from qiskit.circuit import QuantumCircuit
+        from qiskit.compiler import transpile
+
+        backend = FakeKolkata()
+        qc = QuantumCircuit(2)
+        qc.h(0)
+        qc.cx(0, 1)
+        qc.measure_all()
+        backend.run(tramspile(qc, backend, optimization_level=3))
```

Comment:
```suggestion
        backend.run(transpile(qc, backend, optimization_level=3))
```

"trams pile" lol
![pacific-electric-red-cars-awaiting-destruction-44dd9e-1024](https://user-images.githubusercontent.com/5968590/161635419-17c43dba-8896-4649-8c19-6c045b8ea9e7.jpg)

Reply:
Lol, I kind of want to put that picture in the docs somewhere now

Reply:
I got it from some stock image site that was forbidding right-click>save (after an embarrassingly long search for a joke!), so unfortunately it's probably licensed and unusable :(